src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -93,22 +93,38 @@ export class CreateClusterComponent implements OnDestroy {
   }
 
   onSubmit() {
-    forkJoin(this.observables)
-      .pipe(
-        finalize(() =>
-          this.clusterService.updateStatus('POST_INSTALLED').subscribe(() => {
-            this.notificationService.show(
-              NotificationType.success,
-              $localize`Cluster expansion was successful`
-            );
-            this.router.navigate(['/dashboard']);
-          })
-        )
-      )
-      .subscribe({
-        error: (error) => error.preventDefault()
+    this.hostService.list().subscribe((hosts) => {
+      hosts.forEach((host) => {
+        const index = host['labels'].indexOf('_no_schedule', 0);
+        if (index > -1) {
+          host['labels'].splice(index, 1);
+          this.observables.push(this.hostService.update(host['hostname'], true, host['labels']));
+        }
       });
 
+      forkJoin(this.observables)
+        .pipe(
+          finalize(() =>
+            this.clusterService.updateStatus('POST_INSTALLED').subscribe(() => {
+              this.notificationService.show(
+                NotificationType.success,
+                $localize`Cluster expansion was successful`
+              );
+              this.router.navigate(['/dashboard']);
+            })
+          )
+        )
+        .subscribe({
+          error: (error) => error.preventDefault()
+        });
+    });
+
+    this.driveGroup = this.wizardStepService.sharedData;
+    if (this.driveGroup) {
+      const user = this.authStorageService.getUsername();
+      this.driveGroup.setName(`dashboard-${user}-${_.now()}`);
+      this.driveGroups.push(this.driveGroup.spec);
+    }
```

Comment:
I'll remove these changes once https://github.com/ceph/ceph/pull/43041 is merged. this is just here to make the dashboard cephadm job pass.